### ISSUE TITLE
Suggest simpilifcation for grep -v pipeline

### ISSUE THREAD

**Issue by bje-** [koalaman/shellcheck#1219]
#### For new checks and feature suggestions
- [X] shellcheck.net (i.e. the latest commit) currently gives no useful warnings about this
- [X] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh

#!/bin/bash

grep -v string1 FILE | grep -v string2 | grep -v string3 | grep -v string4 | grep -v string5
```

#### Here's what shellcheck currently says:

No issues detected.

#### Here's what I wanted or expected to see:

Shellcheck should suggest eliminating the long pipeline and replacing with a single command like `grep -v -e string1 -e string2 -e string3 -e string4 -e string5`.

**Comment by Dmole** [comment]
grep -vE "string[1-5]|etc"

would by my preference.

**Comment by bje-** [comment]
Sure, even better.

**Comment by Nightfirecat** [comment]
@Dmole Let's say, for the sake of argument, that the regexes are unique and distinct and cannot be compacted like that. `grep -v -e foo -e bar -e baz`
